### ISSUE TITLE
Dev Server does not work

### ISSUE THREAD

**Issue by dmoetech** [sveltejs/sapper#1438]
**Describe the bug**

I can not reload my dev server once it started. It just loads forever. I also can not get to another page via links. 

**Logs**
Please include browser console and server logs around the time this bug occurred.

No error messages in console or anywhere else.

**To Reproduce**
Please make sure you've tested with the latest versions of Svelte and Sapper.

My project:
https://github.com/dmoetech/website3948

git clone https://github.com/dmoetech/website3948
cd website3948
npm install
npm run dev

Just try to use the page you'll see it immediately

To help us help you, if you've found a bug please consider the following:

* If possible, we recommend creating a small repo that illustrates the problem.
* Reproductions should be small, self-contained, correct examples – http://sscce.org.

Occasionally, this won't be possible, and that's fine – we still appreciate you raising the issue. But please understand that Svelte is run by unpaid volunteers in their free time, and issues that follow these instructions will get fixed faster.

**Expected behavior**

The dev server to work.

**Stacktraces**
If you have a stack trace to include, we recommend putting inside a `<details>` block for the sake of the thread's readability:

No error messages or console logs or anything

<details>
  <summary>Stack trace</summary>

  Stack trace goes here...
</details>

**Information about your Sapper Installation:**

- The output of `npx envinfo --system --npmPackages svelte,sapper,rollup,webpack --binaries --browsers`

  System:
    OS: Linux 5.4 Ubuntu 20.04.1 LTS (Focal Fossa)
    CPU: (12) x64 AMD Ryzen 5 3600 6-Core Processor
    Memory: 23.30 GB / 31.37 GB
    Container: Yes
    Shell: 5.0.17 - /bin/bash
  Binaries:
    Node: 10.19.0 - /usr/bin/node
    npm: 6.14.4 - /usr/bin/npm
  Browsers:
    Chrome: 84.0.4147.125
    Firefox: 79.0
  npmPackages:
    rollup: ^2.26.5 => 2.26.5 
    sapper: ^0.28.0 => 0.28.0 
    svelte: ^3.17.3 => 3.24.1 

- Your browser

Chromium latest version

- Your hosting environment (i.e. Local, GCP/AWS/Azure, Vercel/Begin, etc...)

- If it is an exported (`npm run export`) or dynamic application

With export everything works as expected.

**Severity**
How severe an issue is this bug to you? Is this annoying, blocking some users, blocking an upgrade or blocking your usage of Sapper entirely?

It's impossible to develop a website with this behavior.

Note: the more honest and specific you are here the more we will take you seriously.

**Additional context**

The problem started when I added real images.
I also changed the favicons, but I don't think that that;s a problem.

**Comment by Robert-Fairley** [comment]
Was looking through the Sapper issue tracker for another issue, but came across yours.

Cloned and built your repo, running the dev server. Reproduced your error on Windows 10 under WSL in MS Edge 84.

Pressing stop on the continually loading page, however, produced this error:

`TypeError: Failed to fetch dynamically imported module: http://localhost:3002/client/anfrage.f0c99c6f.js`

Hope this helps you move forward.

**Comment by antony** [comment]
@dmoetech Please check the note here: https://sapper.svelte.dev/docs#static and see if it relates to your issue.

**Comment by dmoetech** [comment]
@Robert-Fairley Thanks for your reproduction

@antony My static directory is 14MB big, therefore it is pretty unlikely that it exceeds the cache limit.


I just found out that if I do the following the page works as expected, but it still does not help me:
- delete the index.css from the static directory
- run the server with npx sapper dev

Now I got no styles but pages load as expected. Maybe that helps to solve the issue.
